zproject/prod_settings_template.py

```diff
@@ -168,6 +168,7 @@
 LOCAL_UPLOADS_DIR = "/home/zulip/uploads"
 #S3_AUTH_UPLOADS_BUCKET = ""
 #S3_AVATAR_BUCKET = ""
+#S3_REALM_ICONS_BUCKET = ""
```

Comment:
Yeah, let's keep them in the existing bucket; it's less work to setup that way.  I think it's kinda a mistake that the file uploads and avatar buckets are different buckets already :)